src/applications/vaos/services/patient/index.js

```diff
@@ -378,7 +378,11 @@ export async function fetchFlowEligibilityAndClinics({
     if (!results.patientEligibility.direct.hasRequiredAppointmentHistory) {
       eligibility.direct = false;
       eligibility.directReasons.push(ELIGIBILITY_REASONS.noRecentVisit);
-      recordEligibilityFailure('direct-check-past-visits');
+      recordEligibilityFailure(
+        'direct-check-past-visits',
+        typeOfCare?.id,
+        location?.id,
+      );
```

Comment:
Added type of care and facility location to provide greater context on failed eligibility